pom.xml

```diff
@@ -270,25 +269,20 @@
                             <goal>report</goal>
                         </goals>
                     </execution>
+                    <execution>
+                        <id>report-aggregate</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>report-aggregate</goal>
+                        </goals>
+                    </execution>
                 </executions>
             </plugin>
         </plugins>
 
         <pluginManagement>
             <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-compiler-plugin</artifactId>
-                    <version>3.14.0</version>
-                    <configuration>
```

Comment:
Looks like this configuration is missing now, let's try to move it back (so it does not use defaults)
```

                    <configuration>
                        <release>21</release>
                        <optimize>true</optimize>
                        <debug>true</debug>
                        <encoding>${project.build.sourceEncoding}</encoding>
                    </configuration>

```

After you make the change, could you please run and check if maven uses the params from this configuration in core module (taking into account that this config is present only in parent on pluginManagement level)?